docs/getting-started/istio/index.md

```diff
@@ -30,32 +30,63 @@ spec:
       stableService: rollouts-demo-stable
       trafficRouting:
         istio:
-          virtualService:
-            # Reference to a VirtualService which the controller updates with canary weights
-            name: rollouts-demo-vsvc
+          virtualServices:
+          # One or more virtualServices can be configured
+          # Reference to a VirtualService which the controller updates with canary weights.
+          - name: rollouts-demo-vsvc1
             routes:
             - primary # optional if there is a single route in VirtualService, required otherwise
+          # Reference to a VirtualService which the controller updates with canary weights
+          - name: rollouts-demo-vsvc2
+            routes:
+            - secondary # optional if there is a single route in VirtualService, required otherwise
 ...
 ```
 
-The VirtualService and route referenced in `trafficRouting.istio.virtualService` is required
-to have a HTTP route which splits between the stable and canary Services, referenced in the rollout.
+The VirtualService and route referenced in either `trafficRouting.istio.virtualService` or
+`trafficRouting.istio.virtualServices`. `trafficRouting.istio.virtualServices` helps in adding
+one or more virtualServices unlike `trafficRouting.istio.virtualService` where only single virtualService can be added.
+This is required to have a HTTP route which splits between the stable and canary Services, referenced in the rollout.
 In this guide, those Services are named: `rollouts-demo-stable` and `rollouts-demo-canary` 
 respectively. The weight values for these services used should be initially set to 100% stable, 
 and 0% on the canary. During an update, these values will be modified by the controller.
+If there are multiple VirtualService then weight values for stable and canary service of each VirtualService
+will be modified by the controller.
+
+```yaml
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: rollouts-demo-vsvc1
+spec:
+  gateways:
+  - rollouts-demo-gateway
+  hosts:
+  - rollouts-demo-vsvc1.local
+  http:
+  - name: primary  # Should match spec.strategy.canary.trafficRouting.istio.virtualServices.routes
+    route:
+    - destination:
+        host: rollouts-demo-stable  # Should match spec.strategy.canary.stableService
+      weight: 100
+    - destination:
+        host: rollouts-demo-canary  # Should match spec.strategy.canary.canaryService
+      weight: 0
+
```

Comment:
Changes made to the example override the existing single virtual service cases, so it is a bit hard for beginners to follow.

Instead of mixing them, can we separate these two cases by adding a new example for `trafficRouting.istio.virtualServices`

Reply:
The reason of overriding this existing VS doc is Users are recommended to use "trafficRouting.istio.virtualServices" instead of "trafficRouting.istio.virtualService". Because it provides more flexibility in adding one or more virtualServices. I am still keeping virtualService is to provide the backward compatibility.
Going forward, we can deprecate "trafficRouting.istio.virtualService".

Reply:
Got it. Then I think you need to 1) update the specification doc https://github.com/argoproj/argo-rollouts/blob/master/docs/features/specification.md and 2) put the above explanation in the PR. Thanks.